src/ripple/app/main/GRPCServer.cpp

```diff
@@ -427,9 +428,9 @@ GRPCServerImpl::GRPCServerImpl(Application& app)
     : app_(app), journal_(app_.journal("gRPC Server"))
 {
     // if present, get endpoint from config
-    if (app_.config().exists("port_grpc"))
+    if (app_.config().exists(PORT_GRPC))
```

Comment:
@legleux Apologies for the late reply.

I can include an else section and I can log the details at this juncture. Would that help in addressing your previous concern?

Reply:
I guess it really depends if this section behaves the same as the other ones.
I don't remember now how I came across my previous concern but it was not a very important one.